### DISCUSSION TITLE
Setting initial position

### DISCUSSION THREAD

**Discussion by gsimone** [armandsalle/butter-slider#21] [Q&A]
Hey 👋 thanks for the awesome slider!
I'd like to set the initial slider position, I think I could achieve this by using setRelativePosition with the correct number, but there might be a better way.

eg.
```js
mySlider.setRelativePosition( indexOfSlide * widthOfSlide + margin )
```

**Comment by armandsalle** [comment]
Hey @gsimone! I think it's the good way at the moment, don't forget to handle the responsive of you slider with `window.matchMedia()` you can find the doc here: https://developer.mozilla.org/fr/docs/Web/API/Window/matchMedia

Thank you for your feedback and for using BS on your project 🧈

Feel free to share your creation
